cleaners/vuze.xml

```diff
@@ -18,34 +18,91 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Vuze
+    @url http://www.vuze.com/
+    @os Windows, Linux
+    @cleanerversion v2.0.0
+    @cleanerdate 2019-03-18.2
+    @cleanerby Andrew Ziem (2009-02-15 - 2009-10-04) & Tobias B. Besemer (2018-12-20 - 2019-03-18)
+    @tested ok v???, Windows 7
+    @testeddate ???
+    @testedby https://github.com/Tobias-B-Besemer
+    @note Linux $$ProgramFiles$$ not yet figured out! I will do this later! Tobias.
+
 -->
 <cleaner id="vuze">
-  <label>Vuze / Azureus</label>
+  <label>Vuze/Azureus</label>
   <description>File transfer client</description>
-  <option id="backup_files">
+  <var name="Profile">
+    <value os="windows">%AppData%\Azureus</value>
+    <value os="linux">~/.azureus</value>
+  </var>
+  <var name="ProgramFiles">
+    <!--
+    Windows itself changes %ProgramFiles% for 32-bit processes running in a 64-bit operating system as documented here:
+    https://docs.microsoft.com/en-us/windows/desktop/winprog64/wow64-implementation-details
+    ...so we use %SystemDrive%\Program Files\ & %SystemDrive%\Program Files (x86)\...
+    -->
+    <value os="windows">%SystemDrive%\Program Files\Vuze</value>
+    <value os="windows">%SystemDrive%\Program Files (x86)\Vuze</value>
```

Comment:
According to this [Wikipedia article on Program Files](https://en.wikipedia.org/wiki/Program_Files), the directory name can be localized, so these paths may break on some systems. It's best to use Program Files environment variables instead.

Can Vuze be installed in both the `x86` and "regular"?

Reply:
AFAIK the installer is 32/64-bit... So it's ever in \Program Files\Vuze...

Reply:
Look: https://en.wikipedia.org/wiki/Environment_variable#Default_values
`%ProgramW6432%[20] | Yes | %ProgramW6432% (not supported, not replaced by any value) | %SystemDrive%\Program Files (only in 64-bit version)[20]`

Reply:
OK, it seems to be possible to install in both folders...
https://www.filehorse.com/download-vuze-32/

Reply:
As far as I understand it, if you want that I/we use environment variables, then you must solve this with some code.
https://stackoverflow.com/questions/1283664/python-get-wrong-value-for-os-environprogramfiles-on-64bit-vista
A better docu, but .NET: https://weblog.west-wind.com/posts/2019/Feb/05/Finding-the-ProgramFiles64-Folder-in-a-32-Bit-App

Reply:
I am going to change this to the default multi-value variable $$ProgramFiles$$ . If you seen an issue with let, me know.